parlai/mturk/tasks/dealnodeal/worlds.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+from parlai.mturk.core.worlds import MTurkTaskWorld
+from parlai.core.worlds import validate
+from joblib import Parallel, delayed
+from parlai.tasks.dealnodeal.agents import NegotiationTeacher
+import random
+
+class MTurkDealNoDealDialogWorld(MTurkTaskWorld):
+    """Basic world where each agent gets a turn in a round-robin fashion,
```

Comment:
Can this comment (and the comment for parley) be updated to describe what DealNoDeal is doing?